indexer/packages/postgres/src/query-builders/upsert.ts

```diff
@@ -40,15 +47,14 @@ export default class UpsertQueryBuilder<M extends Model, R = M[]> extends QueryB
             `(${colBindings}) VALUES (${valBindings})`,
             'ON CONFLICT (??) DO',
             `UPDATE SET ${setBindings}`,
-            'WHERE ?? = ?',
+            `WHERE ${idConditionBindings}`,
```

Comment:
iiuc if there are two idColumns like in the case of compliance data, this would be `WHERE ?? = ? AND ?? = ?` right?

Reply:
Yes.